contracts/augur/trading/ZeroXTrade.sol

```diff
@@ -1,178 +1,35 @@
 pragma solidity 0.5.10;
 pragma experimental ABIEncoderV2;
 
-import "ROOT/libraries/math/SafeMathUint256.sol";
-import "ROOT/libraries/ContractExists.sol";
-import "ROOT/libraries/token/IERC20.sol";
-import "ROOT/external/IExchange.sol";
-import "ROOT/trading/IFillOrder.sol";
-import "ROOT/ICash.sol";
-import "ROOT/trading/Order.sol";
-import "ROOT/trading/IZeroXTrade.sol";
-import "ROOT/trading/IAugurTrading.sol";
-import 'ROOT/libraries/Initializable.sol';
-import "ROOT/IAugur.sol";
-import 'ROOT/libraries/token/IERC1155.sol';
-
-
-contract ZeroXTrade is Initializable, IZeroXTrade, IERC1155 {
+import "../libraries/math/SafeMathUint256.sol";
+// import "ROOT/libraries/ContractExists.sol";
+// import "ROOT/libraries/token/IERC20.sol";
+import "../external/IExchange.sol";
+import "../trading/IFillOrder.sol";
+// import "ROOT/ICash.sol";
+// import "ROOT/trading/Order.sol";
+import "../reporting/IShareToken.sol";
+import "../trading/IZeroXTrade.sol";
+import "../trading/IAugurTrading.sol";
+import '../libraries/Initializable.sol';
+import "../IAugur.sol";
+// import 'ROOT/libraries/token/IERC1155.sol';
+
+
+contract ZeroXTrade is Initializable, IZeroXTrade /*, IERC1155 */ {
     using SafeMathUint256 for uint256;
 
-    bool transferFromAllowed = false;
```

Comment:
I'm unclear why any changes happened in this contract at all actually. We need all of this removed logic still on Matic.